lib/contracts/vestingscheme.js

```diff
@@ -0,0 +1,209 @@
+"use strict";
+const dopts = require("default-options");
+
+import { ExtendTruffleContract, ArcTransactionResult, ArcTransactionProposalResult } from "../ExtendTruffleContract";
+const SolidityContract = Utils.requireContract("VestingScheme");
+import { Utils } from "../utils";
+
+export class VestingScheme extends ExtendTruffleContract(SolidityContract) {
+  static async new() {
+    const contract = await SolidityContract.new();
+    return new this(contract);
+  }
+
+
+  /**
+   * see CreateVestingAgreementConfig
+   */
+  _defaultCreateOptions = {
+    beneficiary: undefined,
+    returnOnCancelAddress: undefined,
+    startingBlock: null,
```

Comment:
why is this null and the rest undefined?

Reply:
when dopts sees undefined it throws an exception if the value comes in undefined.  any other value becomes the default.

gotta say, i'm not a big fan of dopts.  it is what was there when i came in.